guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
```

Comment:
<p>In each X-Conv operation N represents <strong>the </strong>number of <strong>the </strong>points in next layer, C represents <strong>the </strong>number of channels, K represents the number of nearest neighbors and D represents <strong>the </strong>dilation rate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>''corrected accordingly'', Thanks.</p>